sdk/client/src/listeners/network-listener/utils/utils.ts

```diff
@@ -48,13 +48,33 @@ const sanitizeRequestResponsePair = (
 		requestResponseSanitizer,
 	}: SanitizeOptions,
 ): RequestResponsePair | null => {
+	// body keys are already be redacted at this point (see getBodyThatShouldBeRecorded)
 	let sanitizedPair: RequestResponsePair | null = pair
 
 	// step 1: pass through user defined sanitizer
 	if (requestResponseSanitizer) {
 		try {
+			try {
```

Comment:
nested trys could be cleaned up into just trying the things that could actually error

Reply:
Done!